home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -2061,7 +2093,7 @@ components:
       discriminator:
         propertyName: type
         mapping:
-          base: '#/components/schemas/ReadCooperationInvitation'
+          cooperation: '#/components/schemas/ReadCooperationInvitation'
```

Comment:
Maybe to add the "cooperationName" field to the ReadCooperationInvitation model, and the "apartmentNumber" field to the ReadApartmentInvitation model? Then there will be clear in which cooperation is a role and in which apartment is ownership.

Reply:
I understand why we need this functional.